awsapilib/controltower/controltower.py

```diff
@@ -433,19 +433,22 @@ def register_organizations_ou(self, name: str, force: bool = False) -> bool:
         return self._register_org_ou_in_control_tower(org_ou)
 
     @validate_availability
-    def create_organizational_unit(self, name):
+    def create_organizational_unit(self, name: str, parent_ou_id: str = '') -> bool:
```

Comment:
It would be more consistent with the design of the parent ou was supported by name and not by id. It would be quite tricky for a user to know the id so we should resolve internally the name to the appropriate id like we do in other places.

Reply:
Makes sense. Changed the method to receive parent_ou_name instead of parent_ou_id.